config/webpack.config.js

```diff
@@ -48,6 +57,8 @@ const devServerConfig = () => {
   return {
     port: args.port || '3808',
     host: args.host || process.env.BIND || 'localhost',
+    publicHost: removePort(args.public) || os.hostname(),
```

Comment:
This now breaks my local development environment (running with `foreman start`, which is the "recommended" way of running from source locally) if i don't set `WEBPACK_OPTS="--public localhost"` since it tries to use my hostname for the dev server.